src/test/groovy/xyz/deseteral/lebkuchenfm/IntegrationSpecification.groovy

```diff
@@ -0,0 +1,25 @@
+package xyz.deseteral.lebkuchenfm
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.test.annotation.DirtiesContext
+import spock.lang.Specification
+
+@SpringBootTest(
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+@DirtiesContext
```

Comment:
Ten @DirtiesContext jest niezbędny tutaj?

Reply:
Na ten moment nie, jak będzie potrzebne to się zastanowię jak to ograć - może jakoś sprytniej niż `DirtiesContext`